.yarn/versions/0ceef01f.yml

```diff
@@ -0,0 +1,2 @@
+undecided:
+  - "@yarnpkg/shell"
```

Comment:
I think that this should probably be minor on shell and the CLI, declining the rest.